components/badge/demo/title.md

```diff
@@ -0,0 +1,37 @@
+---
+order: 7
+title:
+  zh-CN:
```

Comment:
Don't leave it empty (you can use English, I'll update it to Chinese), the build will fail.